src/AspNet.Security.OAuth.Bilibili/AspNet.Security.OAuth.Bilibili.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup> 
+    <PackageValidationBaselineVersion>9.2.0</PackageValidationBaselineVersion>
```

Comment:
There was a release since the original comment, so +1

```suggestion    <PackageValidationBaselineVersion>9.3.0</PackageValidationBaselineVersion>
```